packages/core/src/auth/sso/ssoAccessTokenProvider.ts

```diff
@@ -385,9 +399,8 @@ class AuthFlowAuthorization extends SsoAccessTokenProvider {
     }
 
     override async registerClient(): Promise<ClientRegistration> {
-        const companyName = getIdeProperties().company
         return this.oidc.registerClient({
-            clientName: isCloud9() ? `${companyName} Cloud9` : `${companyName} Toolkit for VSCode`,
+            clientName: this.getClientNameForRegistration(),
```

Comment:
All AWS extensions for IDE "x" will have the same generic client name.

```suggestion
            clientName: isCloud9() ? `${companyName} Cloud9` : `${companyName} IDE Extensions for VSCode`,
```